src/styles/partials/components/_emphasized-image.scss

```diff
@@ -0,0 +1,41 @@
+.dg_card-image {
+  background-color: $white;
+  background-position: right;
+  background-repeat: no-repeat;
+  display: block;
+  position: relative;
+  max-width: 660px;
+  min-height: 285px;
+  border-top: $border-width-large solid $red;
+}
+
+.dg_card-content {
+  display: block;
+  position: absolute;
+  background-color: $white;
+  bottom: -1px;
+  max-width: 410px;
+  width: 100%;
+  font-weight: $font-bold;
+  text-transform: uppercase;
+  font-family: "Montserrat", sans-serif;
+  h1 {
+    color: $black;
+    font-size: $h2-font-size;
+    margin-bottom: 0px;
+  }
+  a {
+    color: $black;
+    text-decoration: none;
+  }
+  a:hover {
+    color: $vibrant-blue;
+  }
+  h4 {
+    color: $gray;
+    font-size: $smallest-font-size;
+    letter-spacing: 10px;
+    margin-bottom: $margin-large;
```

Comment:
i think these two margin lines can be simplified to

```scss
margin: $margin-large auto;
```

Reply:
Done

Reply:
Done